src/devices/Mlx90640/Mlx90640.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp2.1</TargetFrameworks>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="Mlx90640.cs" />
+    <Compile Include="Sampling.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="category.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="README.md" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Drawing.Common" Version="5.0.0" />
```

Comment:
```suggestion
    <PackageReference Include="System.Drawing.Common" Version="$(SystemDrawingCommonPackageVersion)" />
```

Reply:
do you actually use those two in the library? I only saw them in the sample

Reply:
It's only used in the sample, I'm going to delete it in the lib